pootle/runner.py

```diff
@@ -14,11 +14,13 @@
 import sys
 from argparse import SUPPRESS, ArgumentParser
 
+from pootle.core.cache import PERSISTENT_STORES
+from pootle.core.log import cmd_log
```

Comment:
If these imports need to be before the Django ones I will put a comment stating it clearly.